xbmc/video/VideoUtils.cpp

```diff
@@ -182,6 +182,8 @@ void CAsyncGetItemsForPlaylist::GetItemsForPlaylist(const std::shared_ptr<CFileI
 
       if (sortDesc.sortBy == SortByLabel)
         items.ClearSortState();
+      else if (sortDesc.sortBy == SortByDate && sortDesc.sortOrder == SortOrderDescending)
```

Comment:
Can't this be defined before? It looks to have it defined as descending and rewrite the value. Ideally should be set to ascending when it is born

Reply:
Good point. I will revise the change accordingly.

Reply:
Should be better now, @enen92 ...